geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistenceAdvisorImpl.java

```diff
@@ -552,6 +555,10 @@ public boolean checkMyStateOnMembers(Set<InternalDistributedMember> replicates)
         }
       }
     }
+    if (replicates.isEmpty() && splitBrainHappened) {
+      throw new ConflictingPersistentDataException(String
+          .format("Region %s from %s is split-brained from all other members.", regionPath, myId));
```

Comment:
I think "is split-brained" needs to be improved. Base this message on the info one you have above. How about: "Region %s from %s was not part of the same distributed system as the existing online members"

Reply:
fixed.